layouts/partials/scripts.html

```diff
@@ -6,8 +6,6 @@
 <script src="{{ $scripts.Permalink }}" defer></script>
 {{ $player := resources.Get "scripts/youtube-player.js" | resources.Minify }}
 <script src="{{ $player.Permalink }}" defer></script>
-{{ $codemirror := resources.Get "scripts/cm6.ts" | js.Build | resources.Minify }}
```

Comment:
Do we also want to remove `assets/scripts/cm6.ts` from the repo until we re-add it?

Reply:
I don't think so because it won't get bundled or anything without this Get